src/modules/formatter.py

```diff
@@ -533,6 +541,24 @@ def read_set_to_df(self, test=False):
         df['target_pos'] = df.target_sentence_idx.apply(self.sentence_pos)
         (event_sem_activations, target_lang_act, df['lang'], weights_role_concept, weights_role_identif,
          df['event_sem_message']) = zip_longest(*df.message.apply(self.get_message_info))
+
+        if self.priming_experiment and test:
+            df['trans_verb_idx'] = None
+            
+            df['alt_sentence_idx'] = df.apply(lambda x: self.active_to_passive(x.target_sentence_idx, x.message), axis=1)
+            df['alt_sentence'] = df.alt_sentence_idx.apply(self.sentence_from_indices)
+
+            # After generating alt_sentences, delete messages for prime sentences
+            df.iloc[::2,df.columns.get_loc('message')] = np.nan
```

Comment:
FYI, you don't need to import np for small actions because you can use it via pandas :) pd.np.nan in this case